backend/services/src/main/java/com/crimealert/constants/UserConstant.java

```diff
@@ -10,6 +10,9 @@ public class UserConstant {
 	public static final String ENABLE_NOTIFICATIONS = "enableNotifications";
 	public static final String DB = "crimeAlertDB";
 	public static final String COLLECTION = "users";
+	public static final String TOKEN = "token";
+	public static final String OTP_COLLECTION = "oneTimeAuthentication";
```

Comment:
Nit: Should we move TOKEN and OTP_COLLECTION to AuthenticationConstant.java since Authentication has its own table, model, controller and service?